src/speckleifc/importer.py

```diff
@@ -0,0 +1,101 @@
+import time
+from typing import cast
+
+from ifcopenshell.entity_instance import entity_instance
+from ifcopenshell.geom import file
+from ifcopenshell.ifcopenshell_wrapper import TriangulationElement
+
+from speckleifc.converter.data_object_converter import data_object_to_speckle
+from speckleifc.converter.geometry_converter import geometry_to_speckle
+from speckleifc.converter.project_converter import project_to_speckle
+from speckleifc.converter.spatial_element_converter import spatial_element_to_speckle
+from speckleifc.ifc_geometry_processing import create_geometry_iterator
+from speckleifc.ifc_openshell_helpers import get_children
+from speckleifc.render_material_proxy_manager import RenderMaterialProxyManager
+from specklepy.logging.exceptions import SpeckleException
+from specklepy.objects import Base
+
+
+class ImportJob:
```

Comment:
just nitpicking, but unless there is something very specific case against it, i'd recommend using the dataclass way of defining things